app/views/candidate_interface/application_form/_review.html.erb

```diff
@@ -1,91 +1,109 @@
-
-<h2 class="govuk-heading-m govuk-!-font-size-27">Qualifications</h2>
-
-<h3 class="govuk-heading-m">Degree(s)</h3>
-<%= render(CandidateInterface::DegreesReviewComponent.new(application_form: application_form, editable: editable, heading_level: 4, show_incomplete: true, missing_error: missing_error)) %>
-
-<h3 class="govuk-heading-m">Maths GCSE or equivalent</h3>
-<%= render(CandidateInterface::GcseQualificationReviewComponent.new(application_form: @application_form, application_qualification: @application_form.maths_gcse, subject: 'maths', editable: editable, heading_level: 4, missing_error: missing_error, submitting_application: true)) %>
-
-<h3 class="govuk-heading-m">English GCSE or equivalent</h3>
-<%= render(CandidateInterface::GcseQualificationReviewComponent.new(application_form: @application_form, application_qualification: @application_form.english_gcse, subject: 'english', editable: editable, heading_level: 4, missing_error: missing_error, submitting_application: true)) %>
-
-<% if @application_form.science_gcse_needed? %>
-  <h3 class="govuk-heading-m">Science GCSE or equivalent</h3>
-  <%= render(CandidateInterface::GcseQualificationReviewComponent.new(application_form: @application_form, application_qualification: @application_form.science_gcse, subject: 'science', editable: editable, heading_level: 4, missing_error: missing_error, submitting_application: true)) %>
-<% end %>
-
-<h3 class="govuk-heading-m"><%= t('page_titles.other_qualification') %></h3>
-<%= render(CandidateInterface::OtherQualificationsReviewComponent.new(application_form: application_form, editable: editable, heading_level: 4, missing_error: missing_error, submitting_application: true)) %>
-
-<% if @application_form.efl_section_required? %>
-  <h3 class="govuk-heading-m">English as a foreign language assessment</h3>
-  <% if @application_form.english_proficiency.present? %>
-    <%= render(CandidateInterface::ChooseEflReviewComponent.call(@application_form.english_proficiency)) %>
+<section class="govuk-!-margin-bottom-8">
+  <% if !CandidateInterface::EndOfCyclePolicy.can_add_course_choice?(@application_form)%>
+    <h2 class="govuk-heading-m govuk-!-font-size-27 govuk-!-margin-bottom-2"><%= t('page_titles.course_choices') %></h2>
+    <p class="govuk-body">
+      You’ll be able to find courses in <%= days_until_find_reopens %> <%= 'day'.pluralize(days_until_find_reopens) %>
+      (<%= EndOfCycleTimetable.find_reopens.to_s(:govuk_date) %>).
+      You can keep making changes to the rest of your application until then.
+    </p>
   <% else %>
-    <%= render(CandidateInterface::SectionMissingBannerComponent.new(section: 'efl', section_path: candidate_interface_english_foreign_language_start_path, error: missing_error)) %>
+    <h2 class="govuk-heading-m govuk-!-font-size-27">Courses</h2>
```

Comment:
* Use same localised value across all application pages that use this heading.
* The courses heading should only be pluralised on initial application; apply again heading should not be pluralised as you can only have one course choice.